jupyter lab with minimal initial env - use venvs

gerardcl opened the pull request:
pick to 4.x from https://github.com/opendevstack/ods-quickstarters/pull/711

Tasks: 
- [x] Updated documentation in `docs/modules/...` directory
- [x] Ran tests in `<quickstarter>/testdata` directory

tbugfinder commented:
Looks like the PR depends on fixes applied to master, no? Is target branch 4.x intended?

gerardcl commented:
> Looks like the PR depends on fixes applied to master, no? Is target branch 4.x intended?

Hi @tbugfinder , I am not sure if I follow your point. AFAIK master is not a release branch but the main one, hence I fixed first master then since it was possible I also fixed 4.x main release branch. I am not sure though if the issues of the failing checks are related to this, tomorrow I will take a closer look if latest run I've just triggered still fails 👍

braisvq1996 reviewed (approved):
:thumbsup:
